modules/cluster/charts.tf

```diff
@@ -7,6 +7,8 @@ resource "helm_release" "jx-git-operator" {
   version          = "0.0.194"
   create_namespace = true
 
+  values = var.jx_git_operator_values
```

Comment:
Instead of this, we could use 
```
values = [
    "${file("values.yaml")}"
]
```
Example: https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#example-usage---chart-repository
For now, this is ok.

Reply:
This is a bit confusing, since in the mail with the review the text was this:

> Instead of this, we could use
> 
> values = [
>     "${file("values.yaml")}"
> ]
> Example: https://registry.terraform.io/providers/hashicorp/helm/latest/docs/resources/release#example-usage---chart-repository
> For now, this is ok.

So I wondered why the PR wasn't merged...

I looked at the nginx module and I think that solution is both more complicated and give a worse user experience. Typically you only want to set few values. With my solution that can then be done inline in the tf file. If the user wants to put it in a file they case do so and use the file function themselves. Also the nginx solution means that even if the user only want to add one value he still needs to add a file with all the default values.